examples/html-kitchen-sink/.storybook/config.js

```diff
@@ -1,8 +1,22 @@
 import { configure, addParameters, addDecorator } from '@storybook/html';
 import { withA11y } from '@storybook/addon-a11y';
+import withReact from '@storybook/addon-react';
```

Comment:
IMO better to have a separate multi framework example

Reply:
I wanted to make sure that all the HTML examples that we already have are still working